tests/foreman/api/test_provisioning.py

```diff
@@ -98,16 +100,22 @@ def test_rhel_pxe_provisioning(
 
     :parametrized: yes
     """
+    if pxe_loader.vm_firmware == 'bios':
+        pytest.skip('Test cannot be run on BIOS as its not supported')
     host_mac_addr = provisioning_host.provisioning_nic_mac_addr
     sat = module_provisioning_sat.sat
+    # Configure the grubx64.efi image to setup the interface and use TFTP to load the configuration
+    sat.execute("echo -e 'net_bootp6\nset root=tftp\nset prefix=(tftp)/grub2' > pre.cfg")
+    sat.execute(
+        'grub2-mkimage -c pre.cfg -o /var/lib/tftpboot/grub2/grubx64.efi -p /grub2/ -O x86_64-efi efinet efi_netfs efienv efifwsetup efi_gop tftp net normal chain configfile loadenv procfs romfs'
+    )
     host = sat.api.Host(
         hostgroup=provisioning_hostgroup,
         organization=module_sca_manifest_org,
         location=module_location,
         name=gen_string('alpha').lower(),
         mac=host_mac_addr,
         operatingsystem=module_provisioning_rhel_content.os,
-        subnet=module_provisioning_sat.subnet,
```

Comment:
Can we assign it for IPv4 only, and skip this for IPv6?

Reply:
Its not updated

Reply:
We don't need in IPv4 as well, not sure why this was added in the first place. Hostgroup contains subnet already so we should be good